app/services/people/importer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 module People
+  # TODO: rename class, since this is misleading: it is not importing anything fron NOMIS.
```

Comment:
I am renaming this since it's really misleading,  even more considering we added a new call called 
`People::Import`

Reply:
I'll open a different PR for this.